addOns/exim/src/main/java/org/zaproxy/addon/exim/AbstractPopupMenuSaveMessage.java

```diff
@@ -108,8 +114,9 @@ public SaveMessagePopupMenu(String label, MessageComponent messageComponent) {
 
             setButtonStateOverriddenByChildren(false);
 
-            if (!(messageComponent == MessageComponent.REQUEST
-                    || messageComponent == MessageComponent.RESPONSE)) {
+            if (!(messageComponent.equals(MessageComponent.REQUEST)
```

Comment:
I thought enum comparison needed to be .equals so I was just trying to tiddy up. But I should have switched the order of the items so that it wouldn’t NPE.

Reply:
It can use `==`, there's only one value of each enum.